packages/react/src/components/Attachments/Attachment.module.css

```diff
@@ -0,0 +1,18 @@
+.overlay {
+  position: fixed; /* Or absolute, depending on the need */
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  z-index: 1000; /* To overlay everything else */
+  background-color: rgba(0, 0, 0, 0.5);
```

Comment:
We are having this color "#333333B3" for all other modal. So, can you please use this so that we can stay consistent?

Reply:
Good spot! Changed to the same bg color